src/pages/ctrader-login/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { StringParam, useQueryParam } from 'use-query-params'
+import { useSigninAndSignup } from 'components/hooks/use-signin-and-signup'
+import Loading from 'features/components/atoms/loading'
+import CtraderWrapper from 'components/custom/_ctrader-wrapper'
+import Typography from 'features/components/atoms/typography'
+import Layout from 'features/components/templates/layout'
+import { WithIntl } from 'components/localization'
+
+const CtraderLogin = () => {
```

Comment:
Should we show this page for eu users as well?

Reply:
Non of these pages will be accessible from deriv.com links.
Users will be redirected to these pages from cTrader platform.
Since EU users cannot use cTrader, they wont be able to see these pages as well.